src/resources/billing.js

```diff
@@ -0,0 +1,100 @@
+import T from 'proptypes';
+
+export const propTypes = {
+  id: T.string,
+  disabled_at: T.string,
+  object: T.string,
+  name: T.string,
+  last4: T.string,
+  expiration_month: T.string,
+  expiration_year: T.string,
+  brand: T.string,
+  country: T.string,
+  verified: T.bool,
+  bank_account: T.string,
+};
+
+export default (api) =>
+  class Billing {
+    static _name = 'Billing';
+
+    static _url = 'billing';
+
+    static key = 'billing';
+
+    static propTypes = propTypes;
+
+    /**
+     * fund your EasyPost wallet by charging your primary or secondary payment method on file.
```

Comment:
Issue: These docstrings (all of them in this PR) should start with capitols.

Reply:
Atlanta
Boston
Columbus
Denver
Salt Lake City
Albany

... Oh, you meant capitals...

Reply:
capitals* 😛

Reply:
fixed